server/dashboards/preconfigured/bot-framework-datepicker.ts

```diff
@@ -0,0 +1,1146 @@
+/// <reference path='../../../client/@types/types.d.ts'/>
```

Comment:
@CristianPQ - If this is useful, why not change `bot-framework-inst` or `bot-framework`?
Plus, I see you've duplicated `bot_analytics_dashboard` - I know you've been using `bot-framework-inst`, why not use that one instead as a starter?